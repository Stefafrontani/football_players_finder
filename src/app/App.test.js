import React from 'react';
import ReactDOM from 'react-dom';
import ProviderApp from './App';
import renderer from 'react-test-renderer';

describe('ProviderApp Component', () => {
  it('Should render without crashing', () => {
    const div = document.createElement('div');
    ReactDOM.render(ProviderApp, div);
    ReactDOM.unmountComponentAtNode(div);
  });

  it('Snapshot', () => {
    const component = renderer.create(ProviderApp);
    const tree = component.toJSON();

    expect(tree).toMatchSnapshot();
  });
});