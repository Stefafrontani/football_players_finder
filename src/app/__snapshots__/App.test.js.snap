// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ProviderApp Component Snapshot 1`] = `
<div
  className="App"
>
  <h1
    className="fs30 mb12"
  >
     Football Player Finder 
  </h1>
  <section
    className="players-filter-container"
  >
    <div
      className="mb16"
      style={
        Object {
          "width": "100%",
        }
      }
    >
      <form
        className="flex flex-row flex-items-center flex-content-center"
        onSubmit={[Function]}
      >
        <div
          className="mr16 field-container position-relative"
        >
          <input
            id="name"
            name="name"
            onChange={[Function]}
            pattern="[A-Za-z]([A-Za-z]|[' '](?=[A-Za-z]))+|[A-Za-z]"
            type="text"
          />
          <span />
          <label
            className="position-absolute fs15"
            htmlFor="name"
          >
             Name 
          </label>
        </div>
        <div
          className="mr16 field-container position-relative"
        >
          <input
            id="position"
            name="position"
            onChange={[Function]}
            type="text"
          />
          <span />
          <label
            className="position-absolute fs15"
            htmlFor="position"
          >
             Position 
          </label>
        </div>
        <div
          className="mr16 field-container position-relative"
        >
          <input
            id="age"
            max="40"
            min="18"
            name="age"
            onChange={[Function]}
            type="number"
          />
          <span />
          <label
            className="position-absolute fs15"
            htmlFor="age"
          >
             Age 
          </label>
        </div>
        <input
          type="submit"
        />
      </form>
    </div>
    <div
      className="margin-auto table-container"
    >
      <p>
        No data available
      </p>
    </div>
  </section>
</div>
`;
